code/modules/reagents/reagent_containers/blood_pack.dm

```diff
@@ -1,21 +1,26 @@
-/obj/item/reagent_containers/blood
+/obj/item/reagent_containers/glass/blood
 	name = "blood pack"
 	desc = "Contains blood used for transfusion. Must be attached to an IV drip."
```

Comment:
```suggestion
	desc = "Contains blood used for transfusion. Must be attached to an IV drip. It looks like it could be <b>sliced</b> open with something sharp."
```